toolkits/google/arcade_google/utils.py

```diff
@@ -592,6 +592,85 @@ def build_drive_service(auth_token: Optional[str]) -> Resource:  # type: ignore[
     return build("drive", "v3", credentials=Credentials(auth_token))
 
 
+def build_files_list_query(
+    mime_type: str,
+    document_contains: Optional[list[str]] = None,
+    document_not_contains: Optional[list[str]] = None,
+) -> str:
+    query = [f"mimeType = '{mime_type}' and trashed = false"]
+
+    if isinstance(document_contains, str):
+        document_contains = [document_contains]
+
+    if isinstance(document_not_contains, str):
+        document_not_contains = [document_not_contains]
+
+    if document_contains:
+        for keyword in document_contains:
+            name_contains = keyword.replace("'", "\\'")
+            full_text_contains = keyword.replace("'", "\\'")
+            keyword_query = (
+                f"name contains '{name_contains}' or fullText contains '{full_text_contains}'"
```

Comment:
we need to group the left and right side of the `or` inside parentheses, otherwise Google interprets the query in a way that we don't intend. See slack dm for more details

Reply:
good catch! just pushed a fix